### DISCUSSION TITLE
French translations ?

### DISCUSSION THREAD

**Discussion by oiimrosabel** [danilkinkin/buckwheat#106] [Ideas]
I've been checking on Crowdin whether I could help with some French translations, but the section for it is nowhere to be found. 

Is it possible to translate the app in French? I'd be very happy to help !

**Comment by danilkinkin** [comment]
Hello. Of course, I'm glad for your interest of the project.

French [https://crowdin.com/editor/buckwheat/148/en-fr](https://crowdin.com/editor/buckwheat/148/en-fr)
